### ISSUE TITLE
Analogous tags in symCIF/coreCIF

### ISSUE THREAD

**Issue by nautolycus** [COMCIFS/magnetic_dic#75]
Many definitions give a list of "analogous" tags that play a similar role in non-magnetic dictionaries. many symmetry-related ones refer to symCIF, but there is not a DDLm symCIF dictionary. Many of the tags in the original DDL2 symCIF dictionary have been included in recent coreCIF updates - but not all. I've just created a Pull Request that changes the reference to symCIF where there is a current coreCIF entry. Should the others also be imported into coreCIF? The ones that don't seem to have a core equivalent are

_space_group.transform_Pp_abc
_space_group.reference_setting

**Comment by vaitkus** [comment]
I am not sure if this will provide an actionable answer, but a similar issue has been raised in the coreCIF repository (see https://github.com/COMCIFS/cif_core/issues/461).

**Comment by brantonc** [comment]
Yes.  These are in the symCIF dictionary.

From: nautolycus ***@***.***>
Sent: Wednesday, March 13, 2024 11:28 AM
To: COMCIFS/magnetic_dic ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [COMCIFS/magnetic_dic] Analogous tags in symCIF/coreCIF (Issue #75)


Many definitions give a list of "analogous" tags that play a similar role in non-magnetic dictionaries. many symmetry-related ones refer to symCIF, but there is not a DDLm symCIF dictionary. Many of the tags in the original DDL2 symCIF dictionary have been included in recent coreCIF updates - but not all. I've just created a Pull Request that changes the reference to symCIF where there is a current coreCIF entry. Should the others also be imported into coreCIF? The ones that don't seem to have a core equivalent are

_space_group.transform_Pp_abc
_space_group.reference_setting

**Comment by jamesrhester** [comment]
I think the answer here is that there is a small "package of work" to bring the remaining symCIF orphans into coreCIF or multi-block CIF, after which these analogous tags can be described correctly.

**Comment by nautolycus** [comment]
Great. I notice #jamesrhester has now opened an issue on this https://github.com/COMCIFS/MultiBlock_Dictionary/issues/10 in the multiblock dictionary.

**Comment by rowlesmr** [comment]
What's actually different in the symCIF dic wrt core?

It looks like only
- `_space_group.reference_setting`
- `_space_group.transform_Pp_abc`
- `_space_group.transform_Qq_xyz`
- `_space_group_symop.generator_xyz`*
- `_space_group_symop.operation_xyz`*,^




\* although their definitions seem to be identical(ish)? Are they supposed to be aliases?
\^ exists in core, but here because potential alias?

**Comment by vaitkus** [comment]
I think that this should be viewed as a blocking issue given that some human-readable descriptions rely almost fully on data items from the symCIF dictionary. For example, `_parent_space_group.reference_setting` reads:
```
    Analogous tags: Perfectly analogous to
    symCIF:_space_group.reference_setting except that it applies to
    the parent structure.
```
Data item `_space_group.reference_setting` is still not defined in any of the DDLm dictionaries.

Given that both coreCIF and multicoreCIF dictionaries are nearing release, now would be the correct time to add the missing definitions.

**Comment by rowlesmr** [comment]
Is this just adding the definitions to core from https://www.iucr.org/__data/iucr/cif/dictionaries/cif_sym_1.0.1.dic ? I'm not really across any nuance, but could do a straight port from DDL2 to DDLm for

- `_space_group.reference_setting`
- `_space_group.transform_Pp_abc`
- `_space_group.transform_Qq_xyz`


These ones look like they're already in core?
- `_space_group_symop.generator_xyz`
- `_space_group_symop.operation_xyz`

**Comment by jamesrhester** [comment]
I agree we should address this. I believe there are very few remaining, and they were waiting on us sorting out the multi-block approach as the definitions referred to other settings/space groups.

**Comment by vaitkus** [comment]
The missing data names from symCIF were merged into coreCIF (see PR https://github.com/COMCIFS/cif_core/pull/628).

Merging PR #113 should resolve this issue.

**Comment by vaitkus** [comment]
Issue solved by merging #113.
